oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/FieldWithLanguageOption.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  TextField,
+  GroupField,
+  ArrayField,
+  FieldLabel,
+  SelectField,
+} from "react-invenio-forms";
+import { Button, Form, Icon } from "semantic-ui-react";
+
+const emptyRelatedWork = {
+  language: "",
+  title: "",
+};
+
+export const FieldWithLanguageOption = ({
+  fieldPath,
+  label,
+  labelIcon,
+  required,
+  options,
+  showEmptyValue,
+}) => {
+  return (
+    <ArrayField
+      addButtonLabel="Add another language"
+      defaultNewValue={emptyRelatedWork}
+      fieldPath={fieldPath}
+      label={<FieldLabel htmlFor={fieldPath} icon="" label={label} />}
+      required={required}
+      showEmptyValue={showEmptyValue}
+    >
+      {({ arrayHelpers, indexPath }) => {
+        const fieldPathPrefix = `${fieldPath}.${indexPath}`;
+        return (
+          <GroupField optimized>
+            <SelectField
+              clearable
+              fieldPath={`${fieldPathPrefix}.language`}
+              label="Language"
+              optimized
+              options={options.languages}
+              required
+              width={2}
+            />
+
+            <TextField
+              fieldPath={`${fieldPathPrefix}.title`}
+              label="Title"
+              width={9}
+              required
+            />
+
+            <Form.Field>
+              {indexPath > 0 && (
+                <Button
+                  style={{ marginTop: "1.75rem" }}
```

Comment:
Use Invenio's [built-in utility classes to set margins](https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides#L358C1-L362), when possible

Reply:
I have tried this, but it seems that for some reason the invenio class works only with whole values i.e. if I put className="rel-mt-2" that works fine, but if I put "rel-mt-1.75" that does not work at all. Maybe there is some catch I don't know about ?:)

Reply:
@Ducica you're right, for relative units, there are utilities just for 1,2,3,4,5 rems, if fraction is really needed here and there, then you can leave it as is for now.